### PR TITLE
lab-apps(ch05s07):Securing Quarkus Microservices

### PR DESCRIPTION
**Pull request by shatakshi06**

### CODE REVIEW

File: secure-review/apps/quarkus-conference/speaker-dashboard/src/app/components/CreateUserModal.tsx [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+                  </Button>]}>
+                
+                  <Form id="modal-with-form-form">
+                  <FormGroup
+                        label="First Name"
+                        isRequired
+                        fieldId="modal-with-form-form-first-name">
+                        <TextInput
+                            isRequired
+                            type="email"
```
Comment (psolarvi) [inline secure-review/apps/quarkus-conference/speaker-dashboard/src/app/components/CreateUserModal.tsx:50]: Is this ok? (same for the following TextInput)

File: secure-review/apps/quarkus-conference/speaker/.mvn/wrapper/MavenWrapperDownloader.java [hunk 2]
```diff
@@ -0,0 +1,142 @@
+/*
```
Comment (psolarvi) [inline secure-review/apps/quarkus-conference/speaker/.mvn/wrapper/MavenWrapperDownloader.java:1]: We are relying directly on installed Maven, so we can remove completely the .mvn folder.

File: secure-review/solutions/quarkus-conference/speaker/.mvn/wrapper/MavenWrapperDownloader.java [hunk 3]
```diff
@@ -0,0 +1,142 @@
+/*
```
Comment (psolarvi) [inline secure-review/solutions/quarkus-conference/speaker/.mvn/wrapper/MavenWrapperDownloader.java:1]: Same here, we can remove the .mvn directory

File: secure-review/solutions/quarkus-conference/speaker/mvnw [hunk 4]
```diff
@@ -0,0 +1,310 @@
+#!/bin/sh
```
Comment (psolarvi) [inline secure-review/solutions/quarkus-conference/speaker/mvnw:1]: We can remove the mvn wrapper

File: secure-review/solutions/quarkus-conference/speaker/mvnw.cmd [hunk 5]
```diff
@@ -0,0 +1,182 @@
+@REM ----------------------------------------------------------------------------
```
Comment (psolarvi) [inline secure-review/solutions/quarkus-conference/speaker/mvnw.cmd:1]: We can remove the mvn wrapper

File: secure-review/apps/quarkus-conference/speaker/mvnw [hunk 6]
```diff
@@ -0,0 +1,310 @@
+#!/bin/sh
```
Comment (psolarvi) [inline secure-review/apps/quarkus-conference/speaker/mvnw:1]: We can remove the mvn wrapper

File: secure-review/apps/quarkus-conference/speaker/mvnw.cmd [hunk 7]
```diff
@@ -0,0 +1,182 @@
+@REM ----------------------------------------------------------------------------
```
Comment (psolarvi) [inline secure-review/apps/quarkus-conference/speaker/mvnw.cmd:1]: We can remove the mvn wrapper

**Review by psolarvi — APPROVED** [review]

File: secure-review/apps/quarkus-conference/speaker-dashboard/src/app/components/CreateUserModal.tsx (hunk 1 above)
Comment (m-czernek) [reply secure-review/apps/quarkus-conference/speaker-dashboard/src/app/components/CreateUserModal.tsx:50]: Changed to `text`

**Pull request merged**